app/questionnaire/router.py

```diff
@@ -518,15 +518,27 @@ def get_enabled_section_keys(
         self,
     ) -> Generator[ProgressKeyType, None, None]:
         for section_id in self.enabled_section_ids:
-            repeating_list = self._schema.get_repeating_list_for_section(section_id)
-
-            if repeating_list:
+            if repeating_list := self._schema.get_repeating_list_for_section(
+                section_id
+            ):
                 for list_item_id in self._list_store[repeating_list]:
-                    section_key: ProgressKeyType = (section_id, list_item_id)
-                    yield section_key
+                    yield (section_id, list_item_id)
+
+            elif (
+                repeating_blocks_list := self._schema.get_repeating_blocks_list_for_section(
+                    section_id
+                )
+            ) and (
+                self._schema.get_list_collector_type_for_section(section_id)
+                == "ListCollectorContent"
+            ):
```

Comment:
Cause we only yield this section id, list item id keys for repeating if the parent list collector is of Content type (which doesn't track repeating blocks progress otherwise).

Reply:
I'm still not following, which is content special?. It shouldn't be different to normal list collector bar the display?.

Reply:
This one is removed now.